python/samples/kernel-syntax-examples/azure_chat_gpt_with_data_api.py

```diff
@@ -35,7 +35,7 @@
 
 # Create the data source settings
 az_source = AzureAISearchDataSources(**azure_ai_search_settings)
-az_data = AzureDataSources(type="AzureCognitiveSearch", parameters=az_source)
+az_data = AzureDataSources(type="AzureAiSearch", parameters=az_source)
```

Comment:
we have many other places in the codebase that format "AI" like "AzureOpenAI" or "OpenAI." I would prefer to keep the formatting like that, instead of the PascalCase as you are suggesting.